File: src/test/java/de/siegmar/fastcsv/reader/RowHandlerTest.java
package de.siegmar.fastcsv.reader;

import net.codinux.csv.kcsv.reader.CsvRow;
import net.codinux.csv.kcsv.reader.RowHandler;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;

public class RowHandlerTest {

    @Test
    public void test() {
        final RowHandler rh = new RowHandler(1);
        rh.add("foo");
        rh.add("bar");
        final CsvRow csvRow = rh.buildAndReset();

        assertNotNull(csvRow);
        assertEquals("CsvRow[originalLineNumber=1, fields=[foo, bar], comment=false]",
            csvRow.toString());
    }

}

File: src/commonTest/kotlin/net/codinux/csv/kcsv/reader/RowHandlerTest.kt
package net.codinux.csv.kcsv.reader

import kotlin.test.Test
import kotlin.test.assertEquals
import kotlin.test.assertNotNull

class RowHandlerTest {
  @Test
  fun test() {
    val rh = RowHandler(1)
    rh.add("foo")
    rh.add("bar")
    val csvRow = rh.buildAndReset()
    assertNotNull(csvRow)
    assertEquals(
      "CsvRow[originalLineNumber=1, fields=[foo, bar], comment=false]",
      csvRow.toString()
    )
  }
}